packages/Modal/src/controller/Modal.tsx

```diff
@@ -112,22 +113,22 @@ export const Modal = ({
   overrides,
   ...rest
 }: IModalProps): any => {
+  const target = usePortal({ id: MODAL_PORTAL_ID });
```

Comment:
Do need or want an `id`? What happens if there is more than one modal open at the same time would they overwrite each other?

In [admin](https://github.com/thebyte9/blaze/blob/release-core-styles/packages/blaze-admin-ui-utils/src/hooks/use-portal.js) we don't have it at the moment